spec-0007/transition_to_rng.py

```diff
@@ -43,8 +43,10 @@ def _transition_to_rng(old_name, *, position_num=None, end_version=None):
       - If `None` is passed by position and `np.random.seed` has been used,
         the function will change from being seeded to being unseeded.
       - If an integer is passed by position, the random stream will change.
-      - If `np.random` or an instance of `RandomState` is passed by position,
-        an error will be raised.
+      - If an instance of `RandomState` is passed by position, either an
+        error will be raised (pre-NumPy 2.2.0) or the random stream may
```

Comment:
This may be slightly confusing: the bit stream won't change, but the random numbers generated from the bitstream might. So perhaps something like:

> If an instance of `RandomState` is passed by position, an error will be raised (NumPy <2.2.0), or it will be accepted (NumPy >= 2.20) but the random numbers generated may be different from those produced by `RandomState`.

Reply:
Done, thanks.